scripts/prepare_chunks.sh

```diff
@@ -1,13 +1,56 @@
 #!/usr/bin/env bash
 set -e
 
-echo "xg file: ${XG}"
-echo "gbwt file: ${GBWT}"
-echo "gam file: ${GAM}"
-echo "region: ${REGION}"
-echo "output directory: ${OUTDIR}"
+
+while getopts x:h:g:r:o: flag
+do
+    case "${flag}" in
+        x) XG_FILE=${OPTARG};;
+        h) GBWT=${OPTARG};;
+        g) GAM_FILES+=("$OPTARG");;
+        r) REGION=${OPTARG};;
+        o) OUTDIR=${OPTARG};;
+    esac
+done
+
+echo "XG File: " $XG_FILE
+echo "Haplotype File: " $GBWT
+echo "Region: " $REGION
+echo "Output Directory: " $OUTDIR
 
 rm -fr $OUTDIR
 mkdir -p $OUTDIR
 
-vg chunk -x $XG_FILE -a $GAM_FILE -g -c 20 -p $REGION -T -b $OUTDIR/chunk -E $OUTDIR/regions.tsv > $OUTDIR/chunk.vg
+vg_chunk_params="-x $XG_FILE -g -c 20 -p $REGION -T -b $OUTDIR/chunk -E $OUTDIR/regions.tsv"
+
+GAM_FILES_JSON=""
+
+echo "Gam Files:"
+for GAM_FILE in "${GAM_FILES[@]}"; do
+    echo " - $GAM_FILE"
+
+    # Put gam files into array format
+    if [ -z "$GAM_FILES_JSON" ]
+    then
+        GAM_FILES_JSON="$GAM_FILE"
+    else
+        GAM_FILES_JSON="$GAM_FILES_JSON, $GAM_FILE"
+    fi
+
+    vg_chunk_params=" $vg_chunk_params -a $GAM_FILE"
+done
+
+
+# Call vg chunk
+vg chunk $vg_chunk_params > $OUTDIR/chunk.vg
+
+# Cosntruct JSON
+JSON_STRING=$(jq -n \
```

Comment:
This is really good!

We might need to make sure the user actually has `jq` somehow though. Maybe put it in the documentation for making the chunks that you are going to need `jq` installed? Or if we were really cool the script could test and complain instead of erroring out with a command not found.